memphis/memphis.py

```diff
@@ -463,28 +485,31 @@ async def consumer(self, station_name: str, consumer_name: str, consumer_group:
                 raise MemphisError(err_msg)
 
             return Consumer(self, station_name, consumer_name, cg, pull_interval_ms, batch_size, batch_max_time_to_wait_ms, max_ack_time_ms, max_msg_deliveries, start_consume_from_sequence=start_consume_from_sequence, last_messages=last_messages)
-
         except Exception as e:
             raise MemphisError(str(e)) from e
-
-
-class Headers:
-    def __init__(self):
-        self.headers = {}
-
-    def add(self, key, value):
-        """Add a header.
+    async def produce(self, message, station_name: str, producer_name: str, generate_random_suffix: bool =False,  ack_wait_sec: int = 15, headers: Union[Headers, None] = None, async_produce: bool=False, msg_id: Union[str, None]= None):
```

Comment:
params are not in the same order as in the readme please stick to the order as in the readme